src/app.css

```
@@ -57,3 +63,9 @@ a:hover {
     font-size: 1.2rem;
   }
 }
+
+@media screen and (prefers-reduced-motion: no-preference) {
+  html {
+    scroll-behavior: smooth;
+  }
+}
```

Missing blank link at the EOF
```suggestion
}

```